run/grpc-ping/ping.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	pb "github.com/GoogleCloudPlatform/golang-samples/run/grpc-ping/pkg/api/v1"
+	"github.com/golang/protobuf/ptypes"
+)
+
+type pingService struct {
+	pb.UnimplementedPingServiceServer
```

Comment:
Embed or as a normal defined type?

```
type pingService pb.UnimplementedPingServiceServer
```

https://play.golang.org/p/TPhoUO07y_k

Reply:
Embed is how the [grpc-go examples](https://github.com/grpc/grpc-go/blob/aa4eae656c8137762f4cb0065e80178b77acf2dc/examples/features/reflection/server/main.go#L39) show this, but I don't have a preference here. Is a defined type preferred even with a divergence?

Reply:
Keeping it the same as grpc-go examples sounds good to me.